acceptance/tests/resource/file/source_attribute.rb

```diff
@@ -182,43 +189,51 @@ class source_test_module {
   end
 end
 
-new_mod_manifest = <<EOF
-class source_test_module {
-#{mod_manifest_entry}
-}
-EOF
 apply_manifest_on(master, <<-MANIFEST, :catch_failures => true)
   file { '#{mod_manifest_file}':
     ensure => file,
     mode => '0644',
-    content => '#{new_mod_manifest}',
+    content => '#{mod_manifest}',
   }
 MANIFEST
 
 # TODO: Add tests for puppet:// URIs with multi-master/agent setups.
-step "when using a puppet://$server/ URI with a master/agent setup"
+step "When using puppet apply"
 agents.each do |agent|
   step "Setup testing local file sources"
   a_testdir = agent.tmpdir('local_source_file_test')
 
   source = "#{a_testdir}/source_mod/files/source"
-  target = "#{a_testdir}/target"
-
-  on agent, "mkdir -p #{File.dirname(source)}"
-  create_remote_file agent, source, 'Yay, this is the local file.'
+  checksums.each do |checksum_type|
+    target = "#{a_testdir}/target#{checksum_type}"
+    checksum = if checksum_type then "checksum => #{checksum_type}," else "" end
+
+    on agent, "mkdir -p #{File.dirname(source)}"
+    create_remote_file agent, source, 'Yay, this is the local file.'
+
+    step "Using a local file path (#{checksum})"
```

Comment:
it was surprising to me that `checksum` included the trailing comma, which led to unintuitive log messages:

>  Using a local file path (checksum => ctime,)

Can we move the comma to the manifest where the `checksum` is used, or just remove entirely (since no properties/parameters come after `checksum` in the manifests below)?